pro_1.py

```diff
@@ -0,0 +1,92 @@
+import json
+from re import A
+import re
+
+database_file=open("database.txt","r+")
+dict_n=json.load(database_file)
+def loged_in():
+    print("succes")
+
+def login():
+    print("login page")
+    return
+def signup():
+    pattern1="\w{1,}.\w{1,}@\w{1,}.com"
```

Comment:
please use declarative names not just pattern1 pattern2
also .com is not a must in an email